package.json

```diff
@@ -20,7 +20,10 @@
     "json:check:srcdata": "npx ts-node --esm scripts/schema-check-files.ts src/data",
     "json:check:testdata": "npx ts-node --esm scripts/schema-check-files.ts testdata/valid",
     "docs:jsonSchema": "scripts/generate-schema-docs.sh",
-    "prepare:tarball": "npm ci && npm run build && tar -czf dist.tgz -C dist ."
+    "prepare:tarball": "npm ci && npm run build && tar -czf dist.tgz -C dist .",
+    "build:index:timeseries": "npx ts-node --esm scripts/build-timeseries-index.ts",
+    "prebuild": "npm run build:index:timeseries",
+    "predev": "npm run build:index:timeseries"
```

Comment:
These seem to be superfluous duplicates? Maybe the intention is to have different `prebuild` and `predev` steps at some point, but so long as they are identical I think it's a bit confusing to split them out.

Reply:
ah cool, ok